scripts/create_tables.sql

```diff
@@ -1 +1,28 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  dob DATE,
+  specialty CHARACTER VARYING
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  dob DATE,
+  insurance CHARACTER VARYING
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  date TEXT,
```

Comment:
I probably would not use `date` and `time` as my column headers here, since `DATE` and `TIME` are data types for SQL (and SQL is sometimes case-insensitive).